frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -419,13 +421,13 @@ private void serveReleaseSearchResults(ResourceRequest request, ResourceResponse
         serveReleaseSearch(request, response, searchText);
     }
 
+    @SuppressWarnings("Duplicates")
```

Comment:
I'd much prefer if you extracted the duplicate code into a method, rather than suppressing warnings.

Reply:
created `protected List<Release> serveReleaseListBySearchText(String searchText)` in `Sw360Portlet`